### PR TITLE
Update JetBrains.gitignore

### PR DESCRIPTION
**Pull request by OvidiuBujor**
**Reasons for making this change:**

_TODO_

**Links to documentation supporting these rule changes:**

_TODO_

If this is a new template:

 - **Link to application or project’s homepage**: _TODO_

### CODE REVIEW

File: Global/JetBrains.gitignore [hunk 1]
```diff
@@ -44,6 +44,16 @@ cmake-build-*/
 # IntelliJ
 out/
 
+# Eclipse
```
Comment (shiftkey) [inline Global/JetBrains.gitignore]: These rules are already captured in [`Global/Eclipse.gitignore`](https://github.com/github/gitignore/blob/master/Global/Eclipse.gitignore).

File: Global/JetBrains.gitignore [hunk 2]
```diff
@@ -44,6 +44,16 @@ cmake-build-*/
...
+.classpath
+.project
+.settings/
+
+# Intellij
+.idea/
+*.iml
+*.iws
```
Comment (shiftkey) [inline Global/JetBrains.gitignore]: This rule is a duplicate from the existing rule on line 42.

File: Global/JetBrains.gitignore [hunk 3]
```diff
@@ -64,3 +74,7 @@ fabric.properties
 
 # Android studio 3.1+ serialized cache file
 .idea/caches/build_file_checksums.ser
+
+# Maven
```
Comment (shiftkey) [inline Global/JetBrains.gitignore]: Please submit a fresh PR with context into this template if you have Maven-specific changes to propose: [`Maven.gitignore`](https://github.com/github/gitignore/blob/master/Maven.gitignore)

File: Global/JetBrains.gitignore [hunk 4]
```diff
@@ -44,6 +44,16 @@ cmake-build-*/
...
 out/
 
+# Eclipse
+.classpath
+.project
+.settings/
+
+# Intellij
+.idea/
+*.iml
```
Comment (shiftkey) [inline Global/JetBrains.gitignore]: We have a section earlier in the template that talks about additional rules that can be enabled when you are using Maven, which might be relevant:

https://github.com/github/gitignore/blob/7eef17f37c63ce3cbddbdd154ff836f370d0ad70/Global/JetBrains.gitignore#L27-L33

File: Global/JetBrains.gitignore [hunk 5]
```diff
@@ -44,6 +44,16 @@ cmake-build-*/
 # IntelliJ
 out/
 
+# Eclipse
+.classpath
+.project
+.settings/
+
+# Intellij
+.idea/
```
Comment (shiftkey) [inline Global/JetBrains.gitignore]: The start of this file has a set of more explicit exclusions inside `.idea/` for things that should not be version controlled, and this rule here will ensure everything else is excluded. I'm still in favour of committing things that can be shared, based on the guidance from JetBrains here: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839

**Comment by OvidiuBujor** [comment]
Hi Brandan,
Sorry for this pull request I made it by mistake..I’m new at git and still learning..please ignore it.
Thanks!

Trimis de pe iPhone‑ul meu

Pe 7 apr. 2019, la 14:57, Brendan Forster <notifications@github.com> a scris:

**Comment by shiftkey** [comment]
@OvidiuBujor no problem!

I'm going to close this out but feel free to submit other pull requests in the future if you think there are opportunities to improve these templates.

**Pull request closed without merging**